### PR TITLE
Content

### PR DESCRIPTION
**Pull request by smashblu**
This branch focuses on adding content to the web pages. This is stepping closer to a fully usable site, hosted on the web.

Content - Screenshot 1
Content - Screenshot 2
Content - Screenshot 3

### CODE REVIEW

File: design/Content - Screenshot 1.png
Comment (VeysonD) [inline design/Content - Screenshot 1.png:1]: For the screenshots, I meant can you add them into the PR description? You don't have to upload them to the actual site itself.

File: resources/scripts.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+function sticky-header() {
+    if (window.pageYOffset > sticky) {
+        header.classList.add("sticky");
+    } else {
+        header.classList.remove("sticky");
+    }
+}
+
+// Variables and execution for "sticky-header" function
+var header = document.getElementById("nav-bar");
```
Comment (VeysonD) [inline resources/scripts.js:11]: `var` should not be used.  If the variable has no chance of being redeclared then `const` is preferable.

File: resources/scripts.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+var sticky = header.offsetTop;
```
Comment (VeysonD) [inline resources/scripts.js:12]: Same as above

File: resources/scripts.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+window.onscoll = function() {sticky-header()};
```
Comment (VeysonD) [inline resources/scripts.js:13]: perhaps this should be `window.onscroll`?

**Review by VeysonD — CHANGES_REQUESTED** [review]

File: design/Content - Screenshot 1.png
Comment (smashblu) [reply design/Content - Screenshot 1.png:1]: By PR description, are you referring to the README.md?

File: resources/scripts.js (hunk 3 above)
Comment (smashblu) [reply resources/scripts.js:13]: A typo! 😱

File: design/Content - Screenshot 1.png
Comment (VeysonD) [reply design/Content - Screenshot 1.png:1]: In the description of this open PR at the top of this page.  Not in the README.md

**Review by VeysonD — APPROVED** [review]
LGTM

**Pull request merged**